package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "nodejs_mp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@babel/cli": "^7.13.14",
+    "@babel/core": "^7.13.14",
+    "@babel/node": "^7.13.13",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.13.12",
```

Comment:
I suppose `devDependencies` section is a better place for `babel` packages, `babel` doesn't process anything in runtime, it transforms the code beforehand, and only compiled code goes into production.
It might not be so critical for the server side, because server side code isn't delivered to the client devices, but anyway, for consistency it would be better to place everything according to the rules :)

Reply:
I see that `@babel/polyfill` is the only thing that might be used at runtime.